runtime/src/runtime/fvm.rs

```diff
@@ -102,68 +100,6 @@ impl MessageInfo for FvmMessage {
     }
 }
 
-fn handle_send_result(
```

Comment:
Moved with no changes because we now only need to use this logic once (in send_generalized).